<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirObjeto</class>
 <widget class="QWidget" name="IncluirObjeto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir Objeto</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutPrincipal" stretch="1,8,1">
    <property name="spacing">
     <number>19</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="NomeHorizontalLayout" stretch="1,6">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelNome">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditInserirNome">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PontoTab">
       <attribute name="title">
        <string>Ponto</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>19</y>
          <width>611</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
         <item>
          <widget class="QGroupBox" name="groupBoxCoordenadasPonto">
           <property name="title">
            <string>Coordenadas do Ponto</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>591</width>
              <height>89</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayoutXYPonto">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="labelXPonto">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditXPonto">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelYPonto">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditYPonto">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>z:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="RetaTab">
       <attribute name="title">
        <string>Reta</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>611</width>
          <height>491</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <widget class="QGroupBox" name="groupBoxCoordenadasPonto_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>609</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>Coordenadas do Ponto Inicial</string>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget_8">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>561</width>
               <height>89</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayoutPontoInicialReta">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>50</number>
              </property>
              <property name="rightMargin">
               <number>50</number>
              </property>
              <item>
               <widget class="QLabel" name="labelXPontoInicialReta">
                <property name="text">
                 <string>x1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditXPontoInicialReta">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelYPontoInicialReta">
                <property name="text">
                 <string>y1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditYPontoInicialReta">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelZPontoInicialReta">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>z1:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditZPontoInicialReta">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxCoordenadasPontoFinalReta">
           <property name="title">
            <string>Coordenadas do Ponto Final</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>561</width>
              <height>89</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayoutPontoFinalReta">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="labelXPontoFinalReta">
               <property name="text">
                <string>x2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditXPontoFinalReta">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelYPontoFinalReta">
               <property name="text">
                <string>y2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditYPontoFinalReta">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelZPontoFinalReta">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>z2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditZPontoFinalReta">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="WireframeTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Wireframe</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="CurvasTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Curvas</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutCancelOk_6">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonCancel_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOK">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
